src/ucp/api/ucp.h

```diff
@@ -333,6 +335,19 @@ typedef enum {
 
 /**
  * @ingroup UCP_DATATYPE
+ * @brief Generate a reusable identifier for Scatter-gather IOV data type.
```

Comment:
there is no need to specifically create a strided type as "reusable" because we can always make it reusable. this is because ucp_dt_make_strided() returns a datatype handle which can hold a cache.
Also, we defined a "struct" type which can be used instead of "persistent iov", i don't see it in the API..
Another not: ucp_dt_make_strided() is not according to the arch doc, where it was defined as : `ucp_datatype_t ucp_dt_create_strided(size_t extent, ucp_datatype_t datatype, size_t count)`